dotCMS/src/integration-test/java/com/dotcms/auth/providers/saml/v1/DotSamlResourceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.dotcms.auth.providers.saml.v1;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.exception.SecurityException;
+import com.dotcms.saml.IdentityProviderConfigurationFactory;
+import com.dotcms.saml.SamlAuthenticationService;
+import com.dotcms.saml.SamlConfigurationService;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.liferay.portal.model.User;
+import com.liferay.portal.util.WebKeys;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class DotSamlResourceTest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6077896)